src/WHMapper/Pages/Mapper/CustomNode/EveSystemNode.cs

```diff
@@ -26,6 +27,7 @@ public partial class EveSystemNode : ComponentBase
 
         private EveSystemNodeModel _node = null!;
 
+        private HubConnection _hubConnection=null!;
```

Comment:
Remove it, and use hub connection on main OverView page when event OnSystemStatusChanged triggering.
It necessary to send this info in real time to other